### PR TITLE
Refract Implementation

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,25 @@ block the event loop.
...
+    exportSourcemap: true
+}
+protagonist.parse('# My API', options, callback);
+```
+
+The available options are:
+
+Name                   | Description
+---------------------- | ----------------------------------------------------------
+`requireBlueprintName` | Require that the API Blueprint have a title
```
Comment (kylef) [inline README.md]: It might be nice to mention what the default values are for these options.

File: README.md [hunk 2]
```diff
@@ -46,6 +46,25 @@ block the event loop.
...
+`exportSourcemap`      | Enable sourcemap generation
+`type`                 | Set the output structure type as either `ast` or `refract`.
```
Comment (kylef) [inline README.md]: Inconsistency: Full stop on this description but none of the others.

**Comment by zdne** [comment]
@pksunkara I am still missing a description here – would you care to elaborate a bit?

**Comment by zdne** [comment]
@pksunkara why you have removed DO NOT MERGE label if this is waiting for a fix and should not be merged? Also please add a description – I do not want to repeat this anymore.

**Comment by pksunkara** [comment]
Sorry. I was cleaning up some PRs and branches and I accidentally removed the label. I will add the description once the shared/refract branches are ready to be merged.

**Comment by pksunkara** [comment]
@danielgtaylor Please make a PR for changing the README

**Comment by pksunkara** [comment]
I am tagging and releasing protagonist.

**Pull request merged**